unpack.lua

```diff
@@ -0,0 +1,57 @@
+--
```

Comment:
as it is used now both for packer and unpacker.. I think a better name for this module is `pack`.. what do you think?